charts/posthog/values.yaml

```diff
@@ -832,7 +832,8 @@ recordingsBlobIngestion:
     enabled: false
   # -- Pod security context for the plugin-server stack deployment.
   podSecurityContext:
-    enabled: false
+    enabled: true
```

Comment:
is this intentional? that will turn it on for every deploy

Reply:
Is it not intentional? As far as I understand we need the security context for the volume access so without this it wouldn't work?

Reply:
But only for every deploy of the blob ingester?

Reply:
sorry i wasn't paying attention i thought you were setting the top level enabled to on 🤦